packages/editor/src/validator/rules/TraitRules.ts

```diff
@@ -40,20 +49,41 @@ class EventHandlerValidatorRule implements TraitValidatorRule {
         });
       }
 
-      // TODO: util methods has no method schema to check the parameters, so now temporally skip validation
-      if (isExpression(targetId) || targetId === GLOBAL_UTIL_METHOD_ID) {
-        return;
+      if (!targetId) {
+        results.push({
+          message: `Target component Id is empty.`,
+          componentId: component.id,
+          traitType: trait.type,
+          property: `/handlers/${i}/componentId`,
+        });
+        return results;
+      }
+
+      if (targetId === GLOBAL_UTIL_METHOD_ID) {
+        // TODO: util methods has no method schema to check the parameters, so now temporally skip validation
```

Comment:
The util methods also have the parameters specs. Can it read the specs from the registry and validate them?

Reply:
ok, done.